docs/content/en/docs/fleet-manager/rollout/abtest.md

```diff
@@ -331,6 +347,105 @@ If the status shows as `Initialized`, it means the initialization of rollout pro
 
 **Notes**: In the above configuration, we set the `kustomization.interval` to 0s. This disables Fluxcd's periodic synchronization of configurations between the local mirror and cluster. The reason is that Flagger needs to modify the replica counts in Deployments to complete its initialization process. If you are uncertain whether the replicas for all applications in your deployments are set to zero, it is recommended to also set `kustomization.interval` to 0s.
 
+### Using Nginx
```

Comment:
I don't think it should be added in the same usage document. It should add another three level catalogue
`Abtest`, `Blue-Green Deploy` and `Canary`.
Documentation according to different ingresses in three directories